### ISSUE TITLE
Suport x-scheme-handler/file out of the box

### ISSUE THREAD

**Issue by leiserfg** [chmln/handlr#21]
Some apps use it (like telegram-desktop) it should work the same way as using the path without schema.

**Comment by aonphleacs** [comment]
I have been endlessly attempting to fix this problem. I would love to have it just work.
Calibre is another one of those programs that relies on it and it's driving me insane xD

**Comment by chmln** [comment]
@leiserfg @aonphleacs this is added now on master. Should be out in a release soon :tada:

**Comment by SolitudeSF** [comment]
any update?

**Comment by tracure1337** [comment]
I think this is related?

❯xdg-mime query default x-scheme-handler/obsidian 
`obsidian.desktop`


❯ handlr list                                       
```
┌─────────────────────────────────┬──────────────────────────────┐
│ application/pdf                 │ org.gnome.Evince.desktop     │
│ application/x-vmware-vm         │ sublime_text_3.desktop       │
│ application/x-xz-compressed-tar │ org.gnome.FileRoller.desktop │
│ audio/mp4                       │ pragha.desktop               │
│ audio/mpeg                      │ pragha.desktop               │
│ text/plain                      │ sublime_text_3.desktop       │
│ video/x-msvideo                 │ vlc.desktop                  │
└─────────────────────────────────┴──────────────────────────────┘

```

x-scheme-handlers do not appear in the list.

**Comment by chmln** [comment]
Hey guys the fix is out in v0.5

**Comment by aonphleacs** [comment]
This still seems to be an issue for me.
In Calibre 5.14 on Manjaro using Handlr 5.0.
Any logs I might be able to provide to help fix the problem? Or maybe I'm doing something wrong somehow...?
